### ISSUE TITLE
Video Thumbnails Do Not Work With File:// Path

### ISSUE THREAD

**Issue by danbunkr** [dragermrb/capacitor-plugin-video-editor#1]
Hey @dragermrb just wanted to first say thanks for the plugin. I did notice a bug though for generating thumbnails. When you pass a `file://` path into the `thumbnail()` method. It returns an error of "invalid parameters". However it does not happen this for the `edit()` method. After looking into the source I code it seems to be that for edit you are normalizing the file path using `let path = call.getString("path")?.replacingOccurrences(of: "file://", with: "");`. But do not do this on thumbnails. I confirmed that when editing the uri at the javascript layer thumbnails work and am fine with doing this as a work around for now but figured we might want to handle this at the plugin level to give a consistent approach for both methods.

Note the following code works for Javascript:

```
    const { file } = await VideoEditor.thumbnail({
              path: decodeURIComponent(uri.replace('file://', '')),
              width: 600,
              height: 600,
              at: 0,
            });
```

**Comment by dragermrb** [comment]
Thanks, I will try to fix it next week

**Comment by dragermrb** [comment]
Fixed on v4.0.2+
